### PR TITLE
First version on comparison helpers

### LINKED ISSUE #88: Create comparison helpers
- [x] `@eq`
- [x] `@ne`
- [x] `@gt`
- [x] `@gte`
- [x] `@lt`
- [x] `@lte`

### PR DESCRIPTION
**Pull request by smfoote**
Fixes #88

### CODE REVIEW

**Comment by smfoote** [comment]
Needs tests. Don't merge.

File: dist/helpers.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        return main(context);
+      }
+    } else if (elseBody) {
+      return elseBody(context);
+    }
+  }
+};
+
+var helpers = {
+  eq: function eq(context, params, bodies) {
```
Comment (jimmyhchan) [inline dist/helpers.js:28]: you sure you want to keep this API?

**Comment by jimmyhchan** [comment]
Looks good otherwise. We could use some unit tests however

File: dist/helpers.js (hunk 1 above)
Comment (smfoote) [inline dist/helpers.js:28]: No. Do you have suggestions?

**Comment by smfoote** [comment]
Working on tests now. I pushed too quickly.

**Comment by smfoote** [comment]
Now there are tests.

File: dist/helpers.js (hunk 1 above)
Comment (smfoote) [inline dist/helpers.js:28]: I have given this some for thought, and I think this is a good API. The `context` may or may not be a copy of the real context, but that doesn't change the API. The `params` and `bodies` are obviously necessary. Acting on the DOM (e.g. adding event listeners) is possible through access to the `bodies`.

The only other thing that helpers may need is some idea of nesting. I'm not sure whether that would come in the form of an existing param (perhaps `context`), a new param, or some global Tornado state variable. What do you think @jimmyhchan and @prashn64

**Pull request merged**